docs/enhancements/0001_disconnected.md

```diff
@@ -0,0 +1,93 @@
+# Disconnected support for KMM
+
+**Goal**: describe the requirements and implementation ideas for KMM disconnected support
+
+## Introduction: disconnected in OCP
+
+The term "disconnected clusters" refers to clusters that do not have Internet access and thus cannot download publicly
+available container images.
+To support disconnected clusters, a private registry must be used.
+The registry is installed and managed by the cluster administrator.
+
+Red Hat offers [documentation](https://docs.openshift.com/container-platform/4.12/installing/disconnected_install/installing-mirroring-installation-images.html#installing-mirroring-installation-images)
+to mirror the OpenShift system iamges on a private registry.
+Under the hood, the tools used:
+- copy the container images from the public registry on the Internet to the private one;
+- create [`ImageContentSourcePolicy`] objects that configure the mirroring on nodes.
+  [`ImageContentSourcePolicy`] objects configure alternate locations for images specified **by digest only**.
+
+KMM itself can be mirrored like any operator with the usual tools.  
```

Comment:
Wondering if we should not add some note related to DTK , and it's relation to the running RHCOS.

Reply:
Still, DTK is part of OCP, so it should be there with the release payload..